Readme.md

```diff
@@ -4,7 +4,7 @@ A reloading dev server for server side TypeScript projects. Compiles TypeScript
 
 ## Features
 
-- Builds and runs TypeScript really fast (using [`esbuild`](https://github.com/evanw/esbuild))
+- Builds and runs TypeScript really fast (using [`swc`](https://swc.rc))
```

Comment:
In this file I basically just `sed` `esbuild` for `swc` and same for the websites.
I think the readme could eventually benefit from some love, like configuration documentation, etc, but this PR isn't the time for that